templates/partials/preview-question.html

```diff
@@ -0,0 +1,77 @@
+{% from 'components/panel/_macro.njk' import onsPanel %}
+{% from 'macros/helpers.html' import format_paragraphs %}
+
+{% set answers = question.answers %}
+
+<div class="ons-summary__item">
+
+  <h3 class="ons-u-mt-m ons-u-fs-r--b">{{ question.title }}</h3>
+
+  {% set answers_length = answers | length %}
+
+  {% if question.descriptions %}
+    {% set descriptions = question.descriptions %}
+    {% set descriptions_length = descriptions | length %}
+
+    {% for description in descriptions %}
+      <div class="ons-question__description ons-u-mb-m">{{- description | safe -}}</div>
+    {% endfor %}
+  {% endif %}
+
+  {%- if question.guidance -%}
+    {% set contents = question.guidance.contents %}
+    {% call onsPanel({
+      "id": "question-guidance-" ~ question.id,
+      "classes": "ons-u-mb-m"
+    }) %}
+      {% set contents = question.guidance.contents %}
+      {% include 'partials/contents.html' %}
+    {% endcall %}
+  {% endif %}
+
+  {% for answer in answers %}
+
+    {% if loop.last and question.type == "MutuallyExclusive" %}
+      <p><strong>{{ _("Or") }}</strong></p>
+    {% endif %}
+
+    {% if not loop.last or (answer.options and answers_length == 1) %}
+      <p>{{ answer.options_text }}</p>
+    {%  endif %}
+
+    {% if answer.options %}
+      {% if answer.label %}
+        {% set label = answer.label %}
+        <p><strong>{{- label | safe -}}</strong></p>
+      {% endif %}
+      <ul>
+        {% for option in answer.options %}
+          <li>{{ option }}</li>
+        {% endfor %}
+      </ul>
+    {% else %}
+      {% if answer.label %}
+        {% set label = answer.label %}
+        <p><strong>{{- label | safe -}}</strong></p>
+        {% if question.type == "DateRange" %}
+          <p data-qa = "question-type">{{ question.type }}</p>
+        {% endif %}
```

Comment:
I don't get why we have this?

Reply:
It allowed me to get the question type to show on the DateRange question like this. There probably is a better and more generic way of doing this for more different types of question but at the moment I haven't seen any other situations where we need this.

![image (3)](https://user-images.githubusercontent.com/42928680/223704204-2898184e-082b-40a1-9414-63599d763513.png)